### ISSUE TITLE
System.ComponentModel.DataAnnotations works only in PCL

### ISSUE THREAD

**Issue by mfe-** [mfe-/Prism.Validation#4]
reproducible:
Use DataAnnotation for example in a [WPF Project](https://github.com/mfe-/Prism.Validation/blob/7b7bee5f95d1b3165d19a711d33b0fbe88e0d70b/Example/WpfApplication/MainWindowViewModel.cs#L28).

The Method [ValidateProperties](https://github.com/mfe-/Prism.Validation/blob/7b7bee5f95d1b3165d19a711d33b0fbe88e0d70b/Prism.Validation/BindableValidator.cs#L183) will always return true, because the linq query result is empty.

``` csharp
   var propertiesToValidate = _entityToValidate.GetType()
                                                        .GetRuntimeProperties()
                                                        .Where(c => c.GetCustomAttributes(typeof(ValidationAttribute)).Any());
```

Because the type of the DataAnnotation from the wpf project differs from the type `typeof(ValidationAttribute)` of the PCL project which are not equal:

wpf type
`_entityToValidate.GetType().GetRuntimeProperties().ElementAt(1).GetCustomAttributes().FirstOrDefault().GetType().AssemblyQualifiedName` =   `
"System.ComponentModel.DataAnnotations.RequiredAttribute, System.ComponentModel.DataAnnotations, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"` 
pcl type
`typeof(ValidationAttribute).AssemblyQualifiedName` = `System.ComponentModel.DataAnnotations.ValidationAttribute, Portable.DataAnnotations, Version=1.0.0.0, Culture=neutral, PublicKeyToken=6f21be3d5ccff8cd`

Possible solution: Type comparison using Strings (e.g. Type.Name)

**Comment by mfe-** [comment]
Changed Type to String [comparison ](https://github.com/mfe-/Prism.Validation/blob/ea21ed896d36ed843fc86ab5ace5f2d724730886/Prism.Validation/BindableValidator.cs#L184). Anyway [validation evaluates](https://github.com/mfe-/Prism.Validation/blob/ea21ed896d36ed843fc86ab5ace5f2d724730886/Prism.Validation/BindableValidator.cs#L222) to true. Needs to be more investigated.

**Comment by mfe-** [comment]
Fixed with b2f5c53a55b78be19c5a4b747a96032da49b80e0
